README.md

```diff
@@ -1,5 +1,39 @@
 # Northcoders News API
 
+Link to the hosted API- https://nc-news-lhwd.onrender.com
+
+
+Summary and Cloning---------------------------------------------------------------
+
+
+This project creates the backend API endpoints required for the functionality of a front-end website. It suitably handles all data in both a test and developer database, and then also hosts a production database through Elephant SQL and Render.
+
+In order to clone the repository, run 'git clone https://github.com/FredR0101/nc_news.git' in the terminal. You will then have to install multiple dependencies.
+
+
+Dependency Installs---------------------------------------------------------------
+
+
```

Comment:
`npm i` will install all the dependencies listed in the package.json (dev and normal) this should be all that is required after cloning down the repo